core/frontend/src/components/vehiclesetup/overview/LightsInfo.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-card class="ma-2 pa-2">
+    <v-card-title class="align-center">
+      Lights
+    </v-card-title>
+    <v-card-text>
+      <span
+        v-for="light in lights"
+        :key="light"
+        class="d-block"
+      >
+        <v-icon>
+          mdi-dots-horizontal
+        </v-icon>
+        {{ toBoardFriendlyChannel(light) }}
+
+      </span>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import autopilot_data from '@/store/autopilot'
+import autopilot from '@/store/autopilot_manager'
+
+import toBoardFriendlyChannel from './common'
+
+export default Vue.extend({
+  name: 'LightsInfo',
+  computed: {
+    boardType() {
+      return autopilot.current_board?.name
+    },
+    lights() {
+      const servo_params = autopilot_data.parameters.filter(
+        (parameter) => parameter.name.startsWith('SERVO') && parameter.name.endsWith('_FUNCTION'),
+      )
+      const light_params = servo_params.filter((parameter) => parameter.value === 58 || parameter.value === 59)
```

Comment:
It would be nice if we could map such values as enums to be used here over having it hardcoded.

Reply:
Check: https://github.com/bluerobotics/BlueOS-docker/pull/1394